src/main.cpp

```diff
@@ -1818,6 +1808,46 @@ void setup() {
     };
     // when adding parameters, set EDITABLE_VARS_LEN to max of .position
 
+    // Editable values reported to MQTT
+    mqttVars["pidON"] = []{ return &editableVars.at("PID_ON"); };
+    mqttVars["brewSetpoint"] = []{ return &editableVars.at("BREW_SETPOINT"); };
+    mqttVars["brewTempOffset"] = []{ return &editableVars.at("BREW_TEMP_OFFSET"); };
+    mqttVars["steamON"] = []{ return &editableVars.at("STEAM_MODE"); };
+    mqttVars["steamSetpoint"] = []{ return &editableVars.at("STEAM_SETPOINT"); };
+    mqttVars["brewPidDelay"] = []{ return &editableVars.at("PID_BD_DELAY"); };
+    mqttVars["backflushON"] = []{ return &editableVars.at("BACKFLUSH_ON"); };
+    mqttVars["startUsePonM"] = []{ return &editableVars.at("START_USE_PONM"); };
+    mqttVars["startKp"] = []{ return &editableVars.at("START_KP"); };
+    mqttVars["startTn"] = []{ return &editableVars.at("START_TN"); };
+    mqttVars["aggKp"] = []{ return &editableVars.at("PID_KP"); };
+    mqttVars["aggTn"] = []{ return &editableVars.at("PID_TN"); };
+    mqttVars["aggTv"] = []{ return &editableVars.at("PID_TV"); };
+    mqttVars["aggIMax"] = []{ return &editableVars.at("PID_I_MAX"); };
+    mqttVars["steamKp"] = []{ return &editableVars.at("STEAM_KP"); };
+    if (ONLYPID == 0) {
```

Comment:
Could use some blank lines here and there :)
We usually have one before an if-statement.

Reply:
Thanks for the input. I have added some blank lines.